payloads/library/prank/FakeUpdate/payload.txt

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Title:         FakeUpdate
+# Author:        jOte-
+# Version:       1.0
+# Target:        Windows
+#
+# Same as fakeupdate.net but local and single file.
+# Opens IE in kiosk mode and show fake update.
+# ALT-F4 to close
+#
+# Blue   --- Setup
+# Green  --- Done
+
+LED SETUP
+GET SWITCH_POSITION
+LED ATTACK
+ATTACKMODE HID STORAGE
+Q GUI r

```

Comment:
I would add a small delay after setting the Bunny into HID. This is so the target machine has enough time time recognize the Bunny and accepts keystrokes. 

This prevents parts of the payload at the start from being skipped.